routingtable/client_interface.go

```diff
@@ -13,6 +13,7 @@ type RouteTableClient interface {
 	RemovePath(*net.Prefix, *route.Path) bool
 	ReplacePath(*net.Prefix, *route.Path, *route.Path)
 	RefreshRoute(*net.Prefix, []*route.Path)
+	Destroy()
```

Comment:
Same here. What are the semantics? Maybe move this function to an own interface and use interface compositioning?

Reply:
Added comment